### PR TITLE
fixes

### LINKED ISSUE #226: Spacing between contributor and title is too small.
The "y" should not be that close to the emojis.

2021-05-06-161123_942x344_scrot

### PR DESCRIPTION
**Pull request by sounak98**
fix #226 #227 #242

### CODE REVIEW

File: src/components/DocsContent/index.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ const mdxGrommetMap = {
     else
       return (
         <Grommet.Box pad="large">
-          <Grommet.Image {...props} />
+          <Grommet.Image {...props} width="100%" />
```
Comment (lucasvo) [inline src/components/DocsContent/index.js:141]: Can't we just not resize the images at all and leave them at whatever size they normally would be?

Comment (lucasvo) [reply src/components/DocsContent/index.js:141]: this will lead to very weird resizing by default.

**Review by lucasvo — COMMENTED** [review]
Can you please reference the issue # this fixes in the description?

File: src/components/DocsContent/index.js (hunk 1 above)
Comment (sounak98) [reply src/components/DocsContent/index.js:141]: This fixes the issues and anyways this is the width, so it won't cause any problem. Can you check https://centrifuge-docs.netlify.app?

**Comment by sounak98** [comment]
> Can you please reference the issue # this fixes in the description?

I have mentioned the issues in each commit message.

**Pull request merged**